tests/a-framework/parameter_file_2.prm.in

```diff
@@ -0,0 +1 @@
+@SOURCE_DIR@
```

Comment:
@davydden It think preprocessing with `CONFIGURE_FEATURE` is the laziest and most convenient approach. That way every `@VARIABLE@` gets substituted by the contents of the corresponding CMake variable. Give it a try!

Reply:
i will once this is merged! 👍